modules/log/module.go

```diff
@@ -98,6 +114,29 @@ func (m *Module) Act(r *domain.Resource, action string, params map[string]interf
 	return r.Configs, nil
 }
 
+func (m *Module) Log(ctx context.Context, r *domain.Resource, filter map[string]string) (chan domain.LogChunk, error) {
+	var cfg config
+	if err := mapstructure.Decode(r.Configs, &cfg); err != nil {
+		return nil, errors.New("unable to parse configs")
+	}
```

Comment:
@mabdh what do you think of this approach?

Reply:
Yes, this is better!